src/rosdiscover/cli.py

```diff
@@ -20,10 +20,9 @@
 
 def _launch(config: Config) -> SystemSummary:
     logger.info(f"reconstructing architecture for image [{config.image}]")
-    with Interpreter.for_image(config.image,
-                               config.sources,
-                               environment=config.environment
-                               ) as interpreter:
+    with Interpreter.for_config(config) as interpreter:
+        ros_dist = interpreter.system.description.distribution
+        logger.info(f'Detected {ros_dist.ros} version: {ros_dist.name}')
```

Comment:
This looks like leftover debugging code?

Reply:
No. I think it's useful information to output as part of the progress of ROSDiscover.